src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,52 +1,108 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
 public class ArrayList<T> implements List<T> {
 
+    private int defaltCapacity = 10;
+    public Object[] elementData;
+    private int size = 0;
+
+    public ArrayList() {
+        elementData = new Object[defaltCapacity];
+    }
+
+    private void grow() {
+        if (size == elementData.length) {
+            elementData = Arrays.copyOf(elementData, elementData.length * 2);
```

Comment:
 why we increment size in twice?

Reply:
Why not?)
Increment it in 1/2, like in source code?

Reply:
fixed